style.css

```diff
@@ -0,0 +1,55 @@
+main {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+.item {
+  padding: 1rem;
```

Comment:
You can use utility class here, else everything looks good.

Reply:
I'm not using here to keep the structure clean for the landing page